backend/lib/edgehog/campaigns/supervisor.ex

```diff
@@ -42,7 +44,19 @@ defmodule Edgehog.UpdateCampaigns.Supervisor do
   end
 
   case @mix_env do
-    :test -> defp children, do: @base_children
-    _other -> defp children, do: @base_children ++ [Edgehog.UpdateCampaigns.Resumer]
+    :test ->
+      defp children, do: @base_children
+
+    _other ->
+      defp children, do: @base_children ++ [update_campaigns_resumer()]
+
+      defp update_campaigns_resumer do
+        update_campaigns_stream =
+          UpdateCampaign
+          |> Ash.Query.for_read(:read_all_resumable)
+          |> Ash.stream!()
+
+        {Resumer, update_campaigns_stream}
```

Comment:
`Resumer` is not aliased in the file, the whole module name is needed:

```suggestion
        {Edgehog.Campaigns.Resumer, update_campaigns_stream}
```